components/prism-julia.js

```diff
@@ -1,13 +1,16 @@
-Prism.languages.julia= {
+Prism.languages.julia = {
 	'comment': {
 		pattern: /(^|[^\\])#.*/,
 		lookbehind: true
 	},
-	'string': /("""|''')[\s\S]+?\1|("|')(?:\\.|(?!\2)[^\\\r\n])*\2/,
-	'keyword' : /\b(?:abstract|baremodule|begin|bitstype|break|catch|ccall|const|continue|do|else|elseif|end|export|finally|for|function|global|if|immutable|import|importall|in|let|local|macro|module|print|println|quote|return|struct|try|type|typealias|using|while)\b/,
-	'boolean' : /\b(?:true|false)\b/,
-	'number' : /(?:\b(?=\d)|\B(?=\.))(?:0[box])?(?:[\da-f]+\.?\d*|\.\d+)(?:[efp][+-]?\d+)?j?/i,
+	'string': {
+		pattern: /("""|''')[\s\S]+?\1|("|')(?:\\.|(?!\2)[^\\\r\n])*\2/,
```

Comment:
`'''` does not start a multiline string in julia. Single quote is used for characters and for the adjoint postfix operator.

See github's correctish highlighting below:

```julia
A'
A' + B'
```

`'.'` is a valid pattern for a Julia character.

Reply:
Well, mostly valid. Adjoint has precedence.

```julia
julia> A = rand(3,3)
3×3 Array{Float64,2}:
 0.582336  0.46317   0.894981
 0.825888  0.527872  0.115864
 0.893903  0.524644  0.442188

julia> b = rand(3)
3-element Array{Float64,1}:
 0.995133361771265
 0.7350168723165764
 0.4650369009421691

julia> A'b
3-element Array{Float64,1}:
 1.6022412002434918
 1.0928888930591456
 1.1814207351859871

julia> A'b''
3-element Array{Float64,1}:
 1.6022412002434918
 1.0928888930591456
 1.1814207351859871
```

Reply:
Thanks for the insight! I'll fix it.